File: src/main/java/no/nav/aap/api/sts/SystemTokenTjeneste.java
package no.nav.aap.api.sts;


import no.nav.aap.api.rest.Pingable;

import static no.nav.aap.api.util.TokenUtil.BEARER;

public interface SystemTokenTjeneste extends Pingable {
    SystemToken getSystemToken();

    default String bearerToken() {
        return BEARER + getSystemToken().getToken();
    }
}

File: src/main/kotlin/no/nav/aap/api/oppslag/system/SystemTokenTjeneste.kt
package no.nav.aap.api.oppslag.system

import no.nav.aap.health.Pingable
import no.nav.aap.util.StringExtensions.asBearer


interface SystemTokenTjeneste  {
    fun getSystemToken(): SystemToken
    fun bearerToken() = getSystemToken().token.asBearer()
}